etc/profile-m-z/newsboat.profile

```diff
@@ -16,7 +21,12 @@ include disable-passwdmgr.inc
 include disable-programs.inc
 include disable-xdg.inc
 
-mkdir ${HOME}/.newsboat
```

Comment:
The whitelist below can not work without this.

Reply:
.newsboat is for backwards compatibility

Reply:
Only for newer release or for a long time (e.g. the version in debain stable)?

Reply:
never mind, i'm making it too complicated.

Reply:
Newsboat maintainer here :wave: 

> .newsboat is for backwards compatibility

It's not (and I already explained that to you — twice — in https://github.com/newsboat/newsboat/issues/1504). Newsboat supports .newsboat and $XDG_HOME/newsboat equally; users are free to pick whichever directory they prefer.